src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Tests/Functions/HeathChecks/HealthCheckFunctionTests.cs

```diff
@@ -0,0 +1,98 @@
+using System.Net;
+using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Functions.HealthChecks;
+using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Functions.HealthChecks.Dtos;
+using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Functions.HealthChecks.Strategies;
+using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Tests.Builders;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Tests.Functions.HeathChecks;
```

Comment:
The very last commit looks like it's reverted some of the fixes to the namespaces.

`HeathChecks` vs. `HealthChecks